RedBugs_webapp/src/main/java/com/isa/pl/redbugs/controller/RouteController.java

```diff
@@ -17,6 +22,21 @@ public RouteController(RouteService routeService) {
     @GetMapping("/routes")
     public String getRoutes() throws IOException {
         routeService.findAllRoutes();
-        return "routes";
+        return "vehicle_form";
     }
+
+    @GetMapping("/routes/details/{routeId}")
+    public String getRouteDetailsById(@PathVariable("routeId") Long routeId, Model model) throws IOException {
+        Route routeFound = routeService.findRouteById(routeId);
+        model.addAttribute("route", routeFound);
+
+        String pageTitle = "List of stops bus number " + routeId;
```

Comment:
"List of stops for bus number" - chociaż musimy się zdecydować na jeden język, więc może lepiej zrobić aplikację po polsku, jeśli to gdańskie autobusy? W takim razie byłoby "Lista przystanków autobusu numer"

Reply:
Dziękuję za uwagę. Poprawiłem!